daemons/aws_elasticsearch_delete_index.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3.6
+import argparse
+import logging
+from urllib.parse import urlparse, urlencode, parse_qs
+
+import boto3
+from botocore.auth import SigV4Auth
+from botocore.awsrequest import AWSRequest
+from botocore.vendored import requests
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+
+logging.basicConfig(level=logging.WARNING)
+log = logging.getLogger(__name__)
+
+#
+# Delete an index from an AWS Elasticsearch domain.
+#
+# arguments:
+#   -h, --help            show this help message and exit
+#   -d DOMAINNAME, --domainname DOMAINNAME
+#                         Elasticsearch domain name
+#   -i INDEX, --index INDEX
+#                         Name of index to delete or "_all"
+#
+
+# This class is copied from: https://github.com/jmenga/requests-aws-sign on 6/16/17
+# Consider refactoring this into a common utility package if there is need to use this elsewhere.
```

Comment:
You will certainly need this elsewhere to talk to ES. I suggest moving it into `dss/util/__init__.py` and importing it here using the path injection trick:
```
pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
sys.path.insert(0, pkg_root)

from dss.util import AWSV4Sign # noqa
```